### ISSUE TITLE
Contributors reported issue can also be displayed.

### ISSUE THREAD

**Issue by c0dehu1k** [MozillaIndia/leaderboard#11]
issues reported to Mozilla can also be displayed, make sure only valid reports are displayed. i.e. those got the status as New (bugzilla terminology)

**Comment by sudheesh001** [comment]
This is nice, bug reports. What say @sankha93 ?

**Comment by grssam** [comment]
I don't see a logic in the code on how issues reported by a user will be displayed. Do you see any such example ?
If the user reports an issue and assigns it to himself , himself, then its his own fault.

**Comment by ngsankha** [comment]
I think he means to add a new column for reported issues.

**Comment by DronRathore** [comment]
Makes sense to me, if someone hasn't reported the bug how come someone get it fixed so that adds to contribution. As far as assigning to himself/herself, there are plenty I see on bugzilla, who does this but being shack-offed by the bugzilla monitors(that's what there work is) or if not I wont find it as a point to be concerned of, its not a war going on, its a love of contribution that people want to display. A +1 from myside :smile:
